tests/cypress/support/tasks/tenderly/vnet-delete.ts

```diff
@@ -0,0 +1,31 @@
+import type { TenderlyAccount } from './types'
+
+/** Implemented as per https://docs.tenderly.co/reference/api#/operations/deleteVnets */
+export type DeleteVirtualTestnetsOptions = {
+  vnet_ids: string[]
+}
+
+export async function deleteVirtualTestnets({
+  accountSlug,
+  projectSlug,
+  accessKey,
+  ...deleteOptions
+}: TenderlyAccount & DeleteVirtualTestnetsOptions) {
+  const response = await fetch(`https://api.tenderly.co/api/v1/account/${accountSlug}/project/${projectSlug}/vnets`, {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+      'X-Access-Key': accessKey,
+    },
+    body: JSON.stringify(deleteOptions),
+  })
+
+  if (!response.ok) {
+    throw new Error(
```

Comment:
We can reuse `fetchJson` from the prices API

Reply:
Nope, because I need to add the X-Access-Key HTTP header and also need to change the `method`. Now I *could* add header options to `fetchJson` from prices api, but honestly, that function was precisely made for prices api and I don't want to needlessly make it more complex. Nothing wrong with just raw dogging `fetch` imo

Reply:
By the way I kinda need to make this a cy.request() anyway if we want to get rid of tasks